### PR TITLE
Adding Telemetry page to describe Console SM telemetry.

### PR DESCRIPTION
**Pull request by theburi**
## Description
Adding a new page to explain Telemetry collected by Console Self-managed.

## When should this change go live?

- [x] This change is not yet live and should not be merged until 8.5 (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by theburi** [comment]
@christinaausley This is ready to be reviewed and merged.

File: docs/self-managed/console-deployment/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- General Data: This includes information about deployed Camunda components, customer ID, and unique installation identifier.
+- Metrics Data: Captures selected Prometheus metrics available for each component via monitoring endpoint. For example, command counts, metrics related to process instances, CPU load, and memory allocation.
+
+### How to Enable Telemetry
+
+By default, the Telemetry configuration is set to “disabled.” Telemetry can be activated by setting the appropriate configuration in the Camunda 8 Helm Chart or setting the appropriate environment variables in the Console configuration. Please refer to the [Console configuration](./configuration.md) page for more information.
+
+### Telemetry configuration options explained.
+
+When the telemetry configuration is set to `disabled`, the Console will not collect or send any metrics. This feature will not be running.
```
Comment (urbanisierung) [inline docs/self-managed/console-deployment/telemetry.md:31]: Maybe worth adding that this is the default setting. If any other or no value is configured, telemetry is disabled.

**Review by urbanisierung — APPROVED** [review]
lgtm

File: docs/self-managed/console-deployment/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+description: "Console (Self-Managed) telemetry introduction"
+---
+
+At Camunda, we strive to offer an excellent user experience at a high and stable level. On a strictly opt-in basis, we are looking to collect environment and usage data to improve your user experience further. These insights help us understand typical environment setups and product usage patterns and will be used to make informed product improvement decisions that benefit you.
+
+## Design
+
+Camunda 8 Console Self-Managed incorporates a telemetry mechanism designed to periodically report usage data to Camunda. This telemetry operates seamlessly in the background and is activated only when telemetry is explicitly enabled. The data collection mechanism is structured to function without interrupting or degrading the performance of the implemented processes. When enabled, the collected data is sent once every 24 hours via HTTPS.
+
+Once telemetry is enabled, a new telemetry page will display collected information. This page can be used to view component metrics as a simple monitoring interface. Console will collect a subset of metrics, which you can download and inspect anytime.
```
Comment (christinaausley) [inline docs/self-managed/console-deployment/telemetry.md:14]: > Once telemetry is enabled, a new telemetry page will display collected information 

Where is this page located? Is this a UI reference?

File: docs/self-managed/console-deployment/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+### How to enable telemetry
+
+By default, the telemetry configuration is set to `disabled`. Telemetry can be activated by setting the appropriate configuration in the Camunda 8 Helm chart or setting the appropriate environment variables in the Console configuration. Refer to [Console configuration](./configuration.md) for more information.
+
+### Telemetry configuration options explained
+
+When the telemetry configuration is set to `disabled` (default), Console will not collect or send any metrics. This feature will not be running.
+
+When the telemetry configuration is set to `download`, Console collects metrics and shows them on a telemetry page. However, it will not send any information to Camunda automatically. With this option, you can report telemetry to Camunda by downloading this data from the telemetry page and sending it to our team on request.
```
Comment (christinaausley) [inline docs/self-managed/console-deployment/telemetry.md:31]: > sending it to our team on request

How would I go about doing this? The contact page?

File: docs/self-managed/console-deployment/telemetry.md (hunk 2 above)
Comment (theburi) [reply docs/self-managed/console-deployment/telemetry.md:14]: YEs, Console SM has new page called Telemetry. Should I explain is in more details?

Comment (christinaausley) [reply docs/self-managed/console-deployment/telemetry.md:14]: Yes please, and/or add a screenshot 👍

Comment (theburi) [reply docs/self-managed/console-deployment/telemetry.md]: Done

**Comment by christinaausley** [comment]
Sorry I'm being so picky about this, but the screenshot renders pretty blurry. Can we adjust?

Alternatively, we can merge what you have and add the screenshot in another PR.

**Comment by theburi** [comment]
Done.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**